### ISSUE TITLE
systemd reports aukspriv fails to start (timeout)

### ISSUE THREAD

**Issue by robberteggermont** [hautreux/auks#5]
Using the latest auks from github on CentOS-7.1, systemd reports aukspriv fails to start (timeout).
```
systemd[1]: Starting SYSV: Auks aukspriv ccache from keytab scripted daemon...
aukspriv[26963]: Starting aukspriv:[  OK  ]
systemd[1]: PID file /var/run/aukspriv.pid not readable (yet?) after start.
systemd[1]: aukspriv.service operation timed out. Terminating.
systemd[1]: Failed to start SYSV: Auks aukspriv ccache from keytab scripted daemon.
systemd[1]: Unit aukspriv.service entered failed state.
```
I see an aukspriv process running afterwards, but no /var/run/aukspriv.pid.

**Comment by robberteggermont** [comment]
This was fixed with commit 9dfac32.

**Comment by hautreux** [comment]
Thanks.
